src/panel.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <title>Block-facebook</title>
+    <link rel="stylesheet" href="panel.css">
+  </head>
+  <body>
+    <div id="panel-1" class="wrapper">
+      <h4 class="profile heading padding">
+        Facebook Container
+      </h4>
+      <p class="padding">
+        Facebook Container has been installed and is preventing Facebook from tracking you around the web. Anytime you see <img src="fb-ban2.png"> this add-on is blocking Facebook from tracking you.
+      </p>  
+    </div>
+    <div>
+      <button class="button">Next</button>
+    </div>
+
+    <script src="panel.js"></script>
```

Comment:
I was a bit confused here, since the panel.js file looks empty in the diff below.